components/sensor/total_daily_energy.rst

```diff
@@ -38,6 +38,9 @@ Configuration variables:
   to integrate over time.
 - **name** (**Required**, string): The name of the sensor.
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **restore** (*Optional*, boolean): Whether to store the intermediate result on the device so
+  that the value can be restored upon power cycle or reboot.
+  Warning: this option can wear out your flash. Defaults to ``true``.
```

Comment:
I don't think we need to warn about this again, as the `min_save_interval` option already mentions it.
```suggestion
  Defaults to ``true``.
```